vulnfeeds/cpp/main.go

```diff
@@ -272,18 +282,26 @@ func CVEToOSV(CVE cves.CVEItem, repos []string, cache git.RepoTagsCache, directo
 func CVEToPackageInfo(CVE cves.CVEItem, repos []string, cache git.RepoTagsCache, directory string) error {
 	CVEID := CVE.CVE.CVEDataMeta.ID // For brevity.
 	CPEs := cves.CPEs(CVE)
-	CPE, err := cves.ParseCPE(CPEs[0]) // For naming the subdirectory used for output.
-	if err != nil {
-		return fmt.Errorf("[%s]: Can't generate an OSV record without valid CPE data", CVEID)
+	maybeVendorName := "ENOCPE"
```

Comment:
Maybe add a comment for why we can't just use empty string here?

Reply:
Added in 2c87616